com.unity.netcode.gameobjects/Editor/HiddenScriptEditor.cs

```diff
@@ -76,7 +75,7 @@ private void Initialize()
 
             m_ServerAddressProperty = connectionDataProperty.FindPropertyRelative(nameof(UnityTransport.ConnectionAddressData.Address));
             m_ServerPortProperty = connectionDataProperty.FindPropertyRelative(nameof(UnityTransport.ConnectionAddressData.Port));
-            m_OverrideBindIpProperty = connectionDataProperty.FindPropertyRelative(nameof(UnityTransport.ConnectionAddressData.ServerListenAddress));
+            connectionDataProperty.FindPropertyRelative(nameof(UnityTransport.ConnectionAddressData.ServerListenAddress));
```

Comment:
It seems that this last adjusted line is not used... and the property found is not being assigned to anything.
Is there a reason to keep this line?

Reply:
Thanks for catching that. The `m_OverrideBindIpProperty` was never being used anywhere so I pulled it out, but obviously not very thoroughly